server/index.js

```diff
@@ -1,10 +1,21 @@
+/* Model imports */
+require('./models/user');
+
+var keys = require('../config/keys');
 var express = require('express');
 /* We are using a single instance of Express to handle incoming requests */
 var app = express();
 
+/* MongoDB set up */
+var mongoose = require('mongoose');
+mongoose.connect(keys.mongoURI);
+var User = mongoose.model('users');
+
 /* Dummy router example */
-app.get('/', (req, res) => {
-    res.send({ hi: 'there' });
+app.get('/api/test', async (req, res) => {
+    var user = await User.findOne({ Id: 123 });
+
+    res.send({ user });
 });
```

Comment:
This is our main JS file for our backend. We connect to the mongodatabase and set up a single router.

Within that router, we write a query to retrieve an object from our database. We then give that object to the front-end to show.